include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,8 +1050,8 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+2), additional_data);
```

Comment:
Exactly, i would prefer if you keep `std::function` implementations un-touched in this PR. In the current state of the `master` branch of deal.II this particular function uses `fe_degree+2`, so yes, as-is is what i wanted :smile:. Thanks!